extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -63,16 +68,45 @@ export const RTC_ERROR_ENUM = {
 /**
  * @param {string} error
  * @param {string} callout
+ * @param {!Window} win
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampDoc
  * @param {number=} opt_rtcTime
  * @return {!Promise<!rtcResponseDef>}
  * @private
  */
-function buildErrorResponse_(error, callout, opt_rtcTime) {
+function buildErrorResponse_(
+  error, callout, errorReportingUrl, win, ampDoc, opt_rtcTime) {
   dev().warn(TAG, `RTC callout to ${callout} caused ${error}`);
+  if (errorReportingUrl) {
+    sendErrorMessage(error, errorReportingUrl, win, ampDoc);
+  }
   return Promise.resolve(/**@type {rtcResponseDef} */(
     {error, callout, rtcTime: opt_rtcTime || 0}));
 }
 
+/**
+ * @param {string} errorType Uses the RTC_ERROR_ENUM above.
+ * @param {string} errorReportingUrl
+ * @param {!Window} win
+ * @param {!../../../src/service/ampdoc-impl.AmpDoc} ampDoc
+ */
+export function sendErrorMessage(errorType, errorReportingUrl, win, ampDoc) {
```

Comment:
`<amp-analytics>` supports sending user error, that could be another way to support this. 
Is error collecting temporary? or is this a feature?

Reply:
This is a feature of RTC. We wanted a way to do this outside of requiring pubs setup amp-analytics ideally

Reply:
can we insert analytics like what we do with activeView?